src/test/java/api/support/http/BusinessLogicInterfaceUrls.java

```diff
@@ -26,6 +26,10 @@ public static URL isbns(String subPath) {
     return getUrl("/isbn", subPath);
   }
 
+  public static URL move(String subPath) {
+    return getUrl("/inventory/items/move", subPath);
+  }
```

Comment:
Why is a sub path needed?

Reply:
It seems this is redundant method, there is already one defined in [src/test/java/api/support/ApiRoot.java](https://github.com/folio-org/mod-inventory/pull/232/files#diff-91a850ea70ae373f4acd55dd58aba487R40)